README.rst

```diff
@@ -0,0 +1,65 @@
+F5 OpenStack Ansible
+====================
+
+|slack badge|
```

Comment:
I see the new ansible channel on F5OpenStack slack.  Are you also going to create a channel on F5OpenStackDev slack?